lib/dox/formatters/json.rb

```diff
@@ -2,9 +2,9 @@ module Dox
   module Formatters
     class Json < Dox::Formatters::Base
       def format
-        return '' if body.nil? || body.length < 2
-
-        JSON.pretty_generate(JSON.parse(body))
+        JSON.pretty_generate(JSON.parse(body || ''))
```

Comment:
Sorry! I just remembered why I didn't rescue the error:
If the headers are telling the content is JSON, then, if the body is invalid JSON, an error should be raised. This is now a silent fail.

In some Rails version in the tests the body was not nil, I think it was ' ' or something else - but when no params were provided. I can't remember exactly how to reproduce it.